packages/cli/src/commands/watch-deps/index.ts

```diff
@@ -31,16 +31,9 @@ const PACKAGE_BLACKLIST = [
 
 const WATCH_LOCATIONS = ['package.json', 'src', 'assets'];
```

Comment:
add `dev` here too, later, after we add `plugin:serve`, right?

Reply:
dev would only be for serving the plugin itself, shouldn't affect build output of the plugin

Reply:
i thought this function would be used inside `backstage-cli plugin:serve`, whose rendered output does depend on changes in `dev` (if that's even how we solve that one). but i may be conflating things

Reply:
This is for watching other packages though, plugin:serve will have it's own local watch mode based on webpack. The locations are the ones we want to watch inside other packages, such as core, or plugins when we're serving the app.

Reply:
I see, thanks for clarifying!